src/ConnectedVMware/ConnectedVMware.Autorest/examples/Get-AzConnectedVMwareInventoryItem.md

```diff
@@ -0,0 +1,55 @@
+### Example 1: List Inventory Items in a resource group of a vcenter
+```powershell
+Get-AzConnectedVMwareInventoryItem -ResourceGroupName "test-rg" -SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d" -VcenterName "test-vc"
+```
+
+```output
+Name                SystemDataCreatedAt   SystemDataCreatedBy                  SystemDataCreatedByType SystemDataLastModifiedAt SystemDataLastModifiedBy             SystemDataLastModifiedByType Kind                   ResourceGroupName
+----                -------------------   -------------------                  ----------------------- ------------------------ ------------------------             ---------------------------- ----                   -----------------
+resgroup-713957     2/16/2023 3:54:14 PM  ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application             8/1/2023 5:27:42 AM      ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application                  ResourcePool           test-rg
+resgroup-754929     2/16/2023 3:54:14 PM  ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application             8/1/2023 5:27:44 AM      ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application                  ResourcePool           test-rg
+dvportgroup-1153526 2/16/2023 3:54:17 PM  ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application             8/1/2023 5:26:11 AM      ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application                  VirtualNetwork         test-rg
+host-713958         2/16/2023 3:54:17 PM  ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application             8/1/2023 5:26:32 AM      ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application                  Host                   test-rg
+vmtpl-vm-1085854    2/16/2023 3:54:17 PM  ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application             8/1/2023 5:29:46 AM      ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application                  VirtualMachineTemplate test-rg
+datastore-563660    2/16/2023 3:54:17 PM  ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application             10/5/2023 5:15:03 PM     ac9dc5fe-b644-4832-9d03-d9f1ab70c5f7 Application                  Datastore              test-rg
+```
+
+This command list all the inventory items of vcenter `test-vc` in a resource group named `test-rg`.
+
+### Example 2: Get a specific Inventory Item
+```powershell
+Get-AzConnectedVMwareInventoryItem -Name "vm-1528708" -VcenterName "test-vc" -ResourceGroupName "test-rg"
+-SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d"
```

Comment:
```suggestion
Get-AzConnectedVMwareInventoryItem -Name "vm-1528708" -VcenterName "test-vc" -ResourceGroupName "test-rg" -SubscriptionId "204898ee-cd13-4332-b9d4-55ca5c25496d"
```
Here shouldn't be a new line, which will be identified as a comandlet rather than a parameter.